<!DOCTYPE html>
<html lang="en">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <meta charset="utf-8">

    <title>GPS :: Global Pneumococcal Sequencing Project</title>
    <meta name="description" content="">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" href="https://netdna.bootstrapcdn.com/bootstrap/3.1.0/css/bootstrap.min.css">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400italic,600italic,400,300,600' rel='stylesheet' type='text/css'>
    <!-- fonts, icons for bootstrap -->
    <link href="https://netdna.bootstrapcdn.com/font-awesome/4.0.3/css/font-awesome.css" rel="stylesheet">
    <link href="css/styles.min.css" rel="stylesheet">

    <!-- HTML5 Support for IE -->
    <!--[if lt IE 9]>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.2/html5shiv.min.js"></script>
    <![endif]-->

  <!-- Favicon -->
  <link rel="shortcut icon" href="gps.ico">
</head>

<body>

    <!-- Header Starts -->
    <header>
      <div class="container">
        <div class="row">
            <div class="logo">
                <h1><i class=""></i><a href="index.html"><span class="color">GPS</span></a></h1>
                <div class="hmeta">Global Pneumococcal Sequencing Project</div>
            </div>
        </div>
      </div>
    </header>

    <!-- Navigation bar starts -->

    <div class="navbar navbar-default bs-docs-nav top-menu" role="banner">
      <div class="container">
       <div class="navbar-header">
         <button class="navbar-toggle" type="button" data-toggle="collapse" data-target=".bs-navbar-collapse">
             <span class="sr-only">Toggle navigation</span>
             <span class="icon-bar"></span>
             <span class="icon-bar"></span>
             <span class="icon-bar"></span>
         </button>
       </div>

       <nav class="collapse navbar-collapse bs-navbar-collapse" role="navigation">
        <ul class="nav navbar-nav">
            <li class="">
                <a id="home" href="index.html">
                    <!-- <i class="fa fa-home blue"> </i> --> Home
                </a>
            </li>
            <li class="dropdown active">
                <a class="dropdown-toggle" data-toggle="dropdown" href="#">
                     About <b class="caret"></b>
                </a>
                <ul class="dropdown-menu">
                    <li class=""><a id="summary" href="project_outline.html">Project Outline</a></li>
                    <li class=""><a id="summary" href="the_team.html">The Team</a></li>
                    <li class=""><a id="founders" href="partners.html">Project Partners </a></li>
                    <li class=""><a id="substudies" href="substudies.html">Sub-studies</a></li>
                </ul>
            </li>
            <li class="dropdown">
                <a class="dropdown-toggle" data-toggle="dropdown" href="#">Resources <b class="caret"></b></a>
                <ul class="dropdown-menu">
                    <li class=""><a id="resources_overview" href="resources_overview.html">Overview</a></li>
                    <li class=""><a id="sampling_map" href="sampling_map.html">Countries</a></li>
                    <li class=""><a id="GPSC_lineages" href="GPSC_lineages.html">Strains</a></li>
                    <li class=""><a id="serotypes" href="serotypes.html">Serotypes</a></li>
                    <!-- <li class=""><a id="public_database" href="/public_database.html">Database (public)</a></li> -->
                    <li class=""><a id="data_downloads" target="_blank" href="/dataviewer/gps/data">Database</a></li>
                </ul>
            </li>
            <li><a id="publications" href="publications.html">Publications</a></li>
            <li class="dropdown">
              <a class="dropdown-toggle" data-toggle="dropdown" href="#">Training <b class="caret"></b></a>
              <ul class="dropdown-menu">
                  <li class=""><a id="training_browser_based" href="training_drag_and_drop.html">Drag and Drop Tools</a></li>
                  <li class=""><a id="training_command_line" href="training_command_line.html">Command Line</a></li>
              </ul>
          </li>
            <li><a id="contact" href="contact.html">Contact</a></li>
        </ul>
      </nav>
       </div>
    </div>
<!-- Navigation bar ends -->

<div id="content">

<!-- Here goes the content for 'summary' menu item in the top navigation bar -->
<section>
    <div class="container content justify">
        <header>
            <h1><i class="fa fa-pencil-square-o blue"></i> Sub-studies</h1>
        </header>
        <div class="desc">
          <p>Many sub-studies have developed from the GPS project and some of them have already been published in high-quality journals and they are listed in the <a href="publications.html">publication</a> page. Here is a list of on-going sub-studies:</p>

          <!-- start substudies -->

          <blockquote>
          <h3>A Retrospective Study of invasive <em>Streptococcus pneumoniae</em> serotype 2 infections causing meningitis in Bangladesh and Africa</h3>
          <strong>Samir Saha</strong>
          <br>
          <em>The Child Health Research Foundation</em>
          <p>Serotype 2 is currently the most common cause of meningitis in Bangladesh though it is rare elsewhere. This study aims to understand the pathogenicity of meningitis-causing serotype 2 pneumococci in Bangladesh using comparative genomics.</p>
          </blockquote>

          <blockquote>
          <h3>A <em>Streptococcus pneumoniae</em> lineage GPSC5 usually associated with pneumococcal conjugate vaccine (PCV) serotypes is the most common cause of serotype 35B invasive disease in South Africa, following routine use of PCV</h3>
          <strong>Kedibone Ndlangisa, Mignon du Plessis, Stephanie Lo</strong>
          <br>
          <em>The National Institute for Communicable Diseases of South Africa</em>
          <p>Serotype 35B is one of the emerging serotypes in South Africa after PCV13. This study will report the major pneumococcal strains driving the expansion of serotype 35B and place it in a global context.</p>
          </blockquote>

          <blockquote>
          <h3>Changes in population dynamics of pneumococcal carriage in South Africa between PCV7 and PCV13 era</h3>
          <strong>Nida Javaid, Stephanie Lo, Shabir Madhi</strong>
          <br>
          <em>Wellcome Sanger Institute and University of Witwatersrand</em>
          <p>The changes in pneumococcal population causing disease in South Africa after PCVs was previously described, yet the vaccine impact on carriage population is scarce. This study will report the population dynamics of pneumococcal carriage from Soweto and Agincourt and relate it to previous studies in disease population.</p>
          </blockquote>

          <blockquote>
          <h3>Comparing pneumococcal population structure to understand patterns of local strain prevalence and the impact of geographic transmission</h3>
          <strong>Rama Kandasamy, Rebecca Gladstone, Nicholas Croucher, Andrew J. Pollard, Stephen Bentley</strong>
          <br>
          <em>University of Oxford</em>
          <p>Studies consistently demonstrate geographical differences in pneumococcal serotype-specific distribution. This study aims to identify the dynamics of novel strains and transmission events between regions.</p>
          </blockquote>

          <blockquote>
          <h3>Defining the mechanisms and evolution of cotrimoxazole resistance in a global dataset of <i>Streptococcus pneumoniae</i></h3>
          <strong>Jennifer Cornick, Anmol Kiran, Dean Everett</strong>
          <br>
          <em>Malawi-Liverpool-Wellcome Trust Clinical Research Programme</em>
          <p>The relatively low concordance in cotrimoxazole susceptibility between phenotype and genotype highlights our knowledge gap in understanding its resistance mechanisms. This study aims to improve our understanding on this topic using GWAS analysis.</p>
          </blockquote>

          <blockquote>
          <h3>Detect relative contributions of mutation and gene exchange within the major circulating clones</h3>
          <strong>Stephen Bentley, Rebecca Gladstone</strong>
          <br>
          <em>Wellcome Sanger Institute</em>
          <p>It is known that there is a subset of pneumococcal strains or clones that are globally disseminated. This study will make clear how mutation and genetic exchange contribute to the success of a strain.</p>
          </blockquote>

          <blockquote>
          <h3>Detect transmission of successful pneumococcal clones on an international scale</h3>
          <strong>Stephen Bentley, Rebecca Gladstone</strong>
          <br>
          <em>Wellcome Sanger Institute</em>
          <p>There are a handful of successful pneumococcal strains and this study is set out to understand the phylodynamic of predominant pneumococcal clones.</p>
          </blockquote>

          <blockquote>
          <h3>Determination of the genes at the CPS locus and correlation with the backbone genotype</h3>
          <strong>Stephen Bentley, Rebecca Gladstone</strong>
          <br>
          <em>Wellcome Sanger Institute</em>
          <p>Serology allows us to type pneumococci into different serotypes. Using genome data, we will look beyond the genes encoding capsule and characterise its relation to the rest of the genome.</p>
          </blockquote>

          <blockquote>
          <h3>Determine the patterns and mechanisms of antibiotic resistance acquisition in a number of developing countries</h3>
          <strong>Stephen Bentley, Rebecca Gladstone</strong>
          <br>
          <em>Wellcome Sanger Institute</em>
          <p>In many LMICs, antibiotic-selective pressure is strong. Understanding the mechanisms of how pneumococci acquire antibiotic resistance will shed light on development of cutting-edge intervention measures.</p>
          </blockquote>

          <blockquote>
          <h3>Diversity of ribosomal mutations associated with MLS resistance</h3>
          <strong>Paulina Hawkins</strong>
          <br>
          <em>Centers for Disease Control and Prevention</em>
          <p>Macrolide has increasingly been used as empirical treatment for respiratory tract infection. After PCV13, rising macrolide resistance was observed in pneumococci with non-vaccine serotype (<a href="https://www.thelancet.com/journals/laninf/article/PIIS1473-3099(19)30297-X/fulltext">Lo et al 2019</a>). This study will improve our understanding of the genetic diversity of ribosomal mutations related to macrolide resistance in <em>S. pneumoniae</em>.</p>
          </blockquote>   

          <blockquote>
          <h3>Elucidating the genetic background of expanding non-vaccine serotypes in a post-PCV13 carriage population in Cambodia</h3>
          <strong>Sophie Belman, Stephanie Lo, Paul Turner</strong>
          <br>
          <em>Wellcome Sanger Institute and Cambodia-Oxford Medical Research Unit (COMRU), Angkor Hospital for Children</em>
          <p>Cambodia introduced PCV13 in 2015. Its impact, together the increase in non-vaccine serotype, was reported (<a href="https://academic.oup.com/cid/article/70/8/1580/5512979">Turner et al 2020</a>). This study is a sequel to the previous study by elucidating which pneumococcal strains are driving the expansion of non-vaccine serotype in carriage population.</p>
          </blockquote>

          <blockquote>
          <h3>Emergence of a cotrimoxazole resistant serotype 16F lineage GPSC46 in an intensively vaccinated African birth cohort</h3>
          <strong>Felix Dube, Stephanie Lo</strong>
          <br>
          <em>Wellcome Sanger Institute and University of Cape Town</em>
          <p>In a longitudinal carriage study in Drakenstein, serotype 16F appears to be a common serotype and most of them belong to GPSC46. This study combines the GPSC46 isolates from both studies to understand its global distribution and evolution.</p>
          </blockquote>

          <blockquote>
          <h3>Epidemiology of invasive pneumococcal disease in Thailand, 2009-2012</h3>
          <strong>Rebecca Hocknell, David W. Cleary, Somporn Srifeungfung, Stuart Clarke</strong>
          <br>
          <em>University of Southampton and Siam University</em>
          <p>Current understanding of invasive pneumococcal disease in Thailand is very limited, and this sub-study will probably be the first to report the population structure of IPD isolates in this region and served as the baseline data to evaluate the PCV impact once it is implemented nationally.</p>
          </blockquote>

          <blockquote>
          <h3>Evolution of CC63 (PMEN25) pre and post PCV introduction</h3>
          <strong>Paulina Hawkins</strong>
          <br>
          <em>Centers for Disease Control and Prevention</em>
          <p>CC63 belongs to GPSC9 is one of the top globally-spreading strain after PCV13. It is multidrug-resistant. This study will shed light on adaptation and evolution under vaccine-selective pressure.</p>
          </blockquote>

          <blockquote>
          <h3>Genetic diversity and distribution of the pneumococcal ABC transporter proteins and implications on potential protein-based vaccines.</h3>
          <strong>Ebrima Bojang</strong>
          <br>
          <em>Wellcome Sanger Institute</em>
          <p>Bacterial ABC transporter is a class of transmembrane transporters that are important for virulence. This study will improve our understanding in their genetic diversity which will shed light on a conservative target for potential vaccine development.</p>
          </blockquote>

          <blockquote>
          <h3>Genomic insight into the evolution of Streptococcus pneumoniae CC63 in India</h3>
          <strong>Balaji Veeraraghavan</strong>
          <br>
          <em>Christian Medical College</em>
          <p>CC63 is one of the predominant pneumococcal clones in India and this study aims to understand the evolution of CC63 in India during the vaccine introduction.</p>
          </blockquote>

          <blockquote>
          <h3>Genomic surveillance of invasive **</strong><em>Streptococcus pneumoniae</em><strong>** isolates in the periods pre- and post-PCV10 introduction in Brazil</h3>
          <strong>Samanta Cristine Grassi Almeida, Stephanie Lo, Maria Cristina de Cunto Brandileone</strong>
          <br>
          <em>Center of Bacteriology, Adolfo Lutz Institute</em>
          <p>Brazil introduced PCV10 in 2010. This study will report the early impact of vaccine introduction nationwide by describing the changes in serotype, pneumococcal strains and antibiotic resistance.</p>
          </blockquote>

          <blockquote>
          <h3>Global distribution and evolution of major penicillin binding proteins that determine β-lactam resistance in <em>Streptococcus pneumoniae</em></h3>
          <strong>Yuan Li, Benjamin Metcalf, Lesley McGee, Bernard Beall, Stephen Bentley, Rebecca Gladstone</strong>
          <br>
          <em>Centers for Disease Control and Prevention</em>
          <p><a href="https://www.cdc.gov/streplab/index.html">CDC Strep Lab</a> has a strong expertise in using genomic data to predict antibiotic resistance in <em>S. pneumoniae</em>. This project will appreciate the genetic diversity and evolution of the penicillin binding protein, that confer resistance to the first-line antibiotic, β-lactam, in treating pneumococcal disease.</p>
          </blockquote>

          <blockquote>
          <h3>Global distribution and evolution of Streptococcus pneumoniae GPSC3 (PMEN33)</h3>
          <strong>Stephanie Lo</strong>
          <br>
          <em>Wellcome Sanger Institute</em>
          <p>GPS headline paper on <a href="https://www.thelancet.com/journals/laninf/article/PIIS1473-3099(19)30297-X/fulltext">Lancet Infectious Diseases</a> has pointed out that GPSC3 is a major pneumococcal strain expressing non-vaccine serotype after PCV13. Using a global collection, we will investigate the evolutionary history of this important lineage and shed light on its secret of success.</p>
          </blockquote>

          <blockquote>
          <h3>Global distribution and genomic diversity of serotype 5 in context of PCVs</h3>
          <strong>Chrispin Chaguza, Rebecca Gladstone, Stephen Bentley</strong>
          <br>
          <em>Wellcome Sanger Institute</em>
          <p>Serotype 5, as a vaccine target, is an invasive serotype that is included in PCV13. It is common in LMICs in Sub Saharan Africa and Asia, but not developed countries. This study will improve our understanding of the pneumococcal strains expressing serotype 5 and its changes during vaccine introduction.</p>
          </blockquote>


          <blockquote>
          <h3>Global diversity of Streptococcus pneumoniae serotype 3 isolates; changes in the pre and post PCV eras in comparison with the changes observed in a local collection set in Hong Kong</h3>
          <strong>Pak-Leung Ho, Pierra Law</strong>
          <br>
          <em>University of Hong Kong</em>
          <p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6422485/pdf/khvi-15-02-1526555.pdf">Ho <em>et al</em></a> has importantly reported that serotype 3 emerged after PCV13 in Hong Kong and its increase was due to a regional-specific lineage that replaced the common serotype 3 clone CC180. This study will elucidate the circulating serotype 3 strains in Hong Kong and understand this emerging strain’s pathogenicity.</p>
          </blockquote>

          <blockquote>
          <h3>Global emergence and population dynamics of <em>Streptococcus pneumoniae</em> serotype 19A/F CC320 (GPSC1)</h3>
          <strong>Balaji Veeraraghavan</strong>
          <br>
          <em>Christian Medical College</em>
          <p>GPSC1 is one of the major pneumococcal strains in India and this study will investigate its population dynamics in India and place it in global context.</p>
          </blockquote>

          <blockquote>
          <h3>Global genomic surveillance characterises the prominent pneumococcal disease threat in the post PCV era - CC989</h3>
          <strong>Rebecca Gladstone</strong>
          <br>
          <em>University of Oslo</em>
          <p>Emergence of serotype 12F after PCV13 was reported across different continents. CC989 is one of the major clones that contributes to the expansion of serotype 12F, especially in Africa (<a href="https://www.thelancet.com/journals/laninf/article/PIIS1473-3099(19)30297-X/fulltext">Lo et al 2019</a>). This study will characterise this lineage in depth and elucidate its acquisition of antibiotic resistance.</p>
          </blockquote>

          <blockquote>
          <h3>Mitis streptococci in a disguise of S. pneumoniae: what is shared, what is distinct?</h3>
          <strong>Ewa Sadowy</strong>
          <br>
          <em>National Medicines Institute</em>
          <p><em>S. mitis</em> occasionally misidentified as <em>S. pneumoniae</em> in clinical laboratories. This project will improve our understanding of the species boundaries using pan genome analysis.</p>
          </blockquote>

          <blockquote>
          <h3>Modelling the pre/post vaccine distribution of accessory genome loci</h3>
          <strong>Stephen Bentley, Nicholas Croucher</strong>
          <br>
          <em>Wellcome Sanger Institute and Imperial College London</em>
          <p>Around 25% of each pneumococcal genome is composed of non-conserved, or ‘accessory’, genes. This study will use a stochastic population model on them to understand why pneumococcal populations have their observed structure, how they change following vaccination and what the effects of different vaccine formulations would likely be, if introduced in different locations.</p>
          </blockquote>

          <blockquote>
          <h3>Molecular Epidemiology of Pneumococcus Isolated from Invasive Pneumococcal Infection of Bangladeshi Children</h3>
          <strong>Roly Malaka, Stephanie Lo</strong>
          <br>
          <em>The Child Health Research Foundation and Wellcome Sanger Institute</em>
          <p>Bangladesh introduced PCV10 in 2015. This study will report the population genetic structure, serotype distribution and antimicrobial resistance of ~500 invasive pneumococcal disease isolates 2002-2015 and serve as the baseline to evaluate the PCV impact in the future.</p>
          </blockquote>

          <blockquote>
          <h3>Persistent colonisation of 19F in children from South Africa eight years after PCV7 introduction</h3>
          <strong>Courtney Olwagen, Stephanie Lo</strong>
          <br>
          <em>University of Witwatersrand and Wellcome Sanger Institute</em>
          <p>This study aims to elucidate why a vaccine targeted serotype has remained in the carriage population in South Africa using bacterial GWAS.</p>
          </blockquote>

          <blockquote>
          <h3>PneumoCaT2: an investigation into 19A-like cps from serotype 19F pneumococcal isolates</h3>
          <strong>Carmen Sheppard, Stephanie Lo</strong>
          <br>
          <em>Public Health of England and Wellcome Sanger Institute</em>
          <p>PHE developed a WGS-based tool<a href="https://pubmed.ncbi.nlm.nih.gov/27672516/"> PneumoCaT</a> to infer serotype from genome data. This study will harness the global collection in the GPS project to understand a 19F divergent. The knowledge learned will ultimately be integrated into a new version of PneumoCaT.</p>
          </blockquote>

          <blockquote>
          <h3>Population genetic structure, serotype distribution and antibiotic resistance of <em>Streptococcus pneumoniae</em> causing invasive disease in children in Argentina</h3>
          <strong>Paula Gagetti, Stephanie Lo, Alejandra Corso</strong>
          <br>
          <em>National Reference Laboratory in Antimicrobial Resistance and Wellcome Sanger Institute</em>
          <p>Argentina introduced PCV13 in 2012. This study will report the early impact of PCV13 on the disease-causing pneumococcal population in Argentina.</p>
          </blockquote>

          <blockquote>
          <h3>Population genetic structure, serotype distribution and antibiotic resistance of <em>Streptococcus pneumoniae</em> in Nigeria</h3>
          <strong>Stephen Obaro, Stephanie Lo</strong>
          <br>
          <em>University of Nebraska Medical Center and Wellcome Sanger Institute</em>
          <p>Nigeria introduced PCV10 in 2014. Current understanding about the epidemiology of <em>S. pneumoniae</em> is scarce. This study will be the first to describe the pneumococcal population using whole-genome sequencing during the introduction of PCV10.</p>
          </blockquote>

          <blockquote>
          <h3>Population genetic structure, serotype distribution and antibiotic resistance of <em>Streptococcus pneumoniae</em> isolates collected in Moscow in 2011-2015</h3>
          <strong>Екатерина Егорова, Stephanie Lo</strong>
          <br>
          <em>Moscow Research Institute for Epidemiology and Microbiology and Wellcome Sanger Institute</em>
          <p>Russia introduced PCV13 in 2014 but its uptake in 2015 was still low. The findings in this study will be used as a baseline to determine the vaccine effectiveness during the following years when higher levels of immunisation coverage were reached.</p>
          </blockquote>

          <blockquote>
          <h3>Population genomics of mobile genetic elements</h3>
          <strong>Stephen Bentley and Nicholas Croucher</strong>
          <br>
          <em></em>
          <p>Mobile genetic elements (MGEs) are common in pneumococcal genomes. This analysis will be focussed on understanding how MGEs affect the diversification of pneumococci.</p>
          </blockquote>

          <blockquote>
          <h3>Relationships and specific features of ST320/serotype 19A isolates from Poland</h3>
          <strong>Ewa Sadowy</strong>
          <br>
          <em>National Medicines Institute</em>
          <p>Multidrug resistant pneumococci of ST320, usually associated with 19A serotype are observed worldwide and currently play an important role in invasive infections in Poland. This study will look into the Polish ST320 isolates and place them in global context.</p>
          </blockquote>

          <blockquote>
          <h3><em>Streptococcus pneumoniae</em> genomic datasets from an Indian population describing pre-vaccine evolutionary epidemiology using a Whole Genome Sequencing approach</h3>
          <strong>Geetha Nagaraj, Stephanie Lo, KL Ravikumar</strong>
          <br>
          <em>Kempegowda Institute of Medical Sciences Hospital &amp; Research Center and Wellcome Sanger Institute</em>
          <p>India has the highest burden of pneumococcal disease worldwide (<a href="https://www.thelancet.com/journals/langlo/article/PIIS2214-109X(18)30247-X/fulltext">Wahl et al 2018</a>). This study will be the first to understand the pneumococcal population causing disease and colonisation using genome data, and more importantly shed light on how well the current and future vaccines target the circulating pneumococcal strains.</p>
          </blockquote>

          <blockquote>
          <h3>Tracking the competitive evolutionary success of a pneumococcal lineage (PMEN15) as it adapts to co-trimoxazole prophylaxis for HIV in South Africa</h3>
          <strong>Stephanie Lo, Nicholas Croucher</strong>
          <br>
          <em>Wellcome Sanger Institute</em>
          <p>South Africa is one of the countries with a high burden of HIV and co-trimoxazole was once introduced as a prophylaxis for HIV patients. This study will investigate how this antibiotic-selective measure affects the circulating pneumococcal lineage PMEN15.</p>
          </blockquote>

          <blockquote>
          <h3>Using genomics to explain the persistence of <em>Streptococcus pneumoniae</em> serotype 19A and the emergence of a new sub-clade associated with vaccine failure</h3>
          <strong>Mary Corcoran and Rebecca Gladstone</strong>
          <br>
          <em>Children&#39;s Health Ireland at Temple Street and Wellcome Sanger Institute</em>
          <p>As one of the vaccine targets, serotype 19A has persisted and caused disease in Ireland. This study will shed light on this vaccine failure incidences using GWAS analysis.</p>
          </blockquote>

          <blockquote>
          <h3>Using Machine Learning to predict cotrimoxazole resistance levels in <em>Streptococcus pneumoniae</em></h3>
          <strong>Stephanie Lo and Nichole Wheeler</strong>
          <br>
          <em>Wellcome Sanger Institute</em>
          <p>Cotrimoxazole resistance is achieved by stepwise mutations in <em>folA</em> and <em>folP</em> genes. The current prediction based on known mutations highlights the needs for improvement. This study will develop a machine learning model to predict cotrimoxazole MIC using genome sequences.</p>
          </blockquote>

          <blockquote>
          <h3>Validation and improvement of predicting β-Lactam minimum inhibitory concentration of Streptococcus pneumoniae using whole-genome sequencing and PBP typing</h3>
          <strong>Yuan Li, Benjamin Metcalf, Lesley McGee, Bernard Beall, Mignon du Plessis, Anne von Gottberg</strong>
          <br>
          <em>Centers for Disease Control and Prevention</em>
          <p>The current machine learning model for prediction of β-Lactam MIC was built and validated using a pneumococcal dataset from the USA (Li et al <a href="https://mbio.asm.org/content/7/3/e00756-16">2016</a> and <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5558719/">2017</a>). This study will validate and improve the prediction model using genome data from South Africa.</p>
          </blockquote>

          <!-- end substudies -->

      </div>
    </div>
</section>

<div id="modal"></div>
<footer>
  <div class="container">
      <div class="row copy">
        <div class="col-lg-7 col-md-7 col-sm-12 copy-left">
          <span>GPS &copy; 2016</span> -
          <a href="cookiespolicy.html" onclick="">Cookies policy</a> | <a href="legal.html">Terms &amp; Conditions.</a>
        </div>
        <div class="col-lg-5 col-md-5 col-sm-12 copy-right">
          This site is hosted by the <a href="https://www.sanger.ac.uk/">Wellcome Trust Sanger Institute</a>
        </div>
      </div>
  </div>

  <div class="clearfix"></div>
</footer>
    <script type="text/javascript" src="https://code.jquery.com/jquery-latest.min.js"></script>
    <script src="https://netdna.bootstrapcdn.com/bootstrap/3.1.0/js/bootstrap.min.js"></script>
    <script src="js/custom.min.js"></script>
    <script type="text/javascript" src="/zxtm/piwik2.js"></script>
    <script type="text/javascript" src="https://www.google.com/jsapi"></script>
    <script type="text/javascript">
      /***** To show chart in the summary page ****/
      google.load('visualization', '1', {packages: ['corechart']});
      google.setOnLoadCallback(drawChart);
    </script>
</body>
</html>
